src/XMakeCommandLine/app.config

```diff
@@ -67,6 +67,7 @@
         <property name="MSBuildToolsRoot" value="$(VsInstallRoot)\MSBuild" />
         <property name="MSBuildExtensionsPath" value="$(VsInstallRoot)\MSBuild" />
         <property name="MSBuildExtensionsPath32" value="$(VsInstallRoot)\MSBuild" />
+        <property name="TargetFrameworkRootPathSearchPathsWindows" value="$(VsInstallRoot)\Common7\IDE\ReferenceAssemblies\Microsoft\Framework" />
```

Comment:
Why define these here with OS specific names and not reuse the current fallback infrastructure via the `projectImportSearchPaths` node? Using that one, you'd define the same `TargetFrameworkRootPathSearchPaths` under each OS